packages.yaml

```diff
@@ -172,6 +172,7 @@ packages:
       - tcl@8.6.6
       - tk@8.6.6 ^tcl@8.6.6
       - mercurial@4.1.2 ^python@2.7.13
+      - tophat@2.1.1
```

Comment:
This will bring in `boost`. As we don't want to have a lot of duplicates, this spec must be more specific so that  it reuses the boost installation that is already on the clusters. Depends also on `bowtie2`.